installer/scripts/service_control

```diff
@@ -93,6 +102,7 @@ start_omsagent_process()
 
 start_omsagent()
 {
+    exit_if_agent_not_onboarded
```

Comment:
On doing so, both "enable" and "start" would have the exact same steps:
1. Register the service if onboarded and not registered.
2. Start the service
Did you want "enable" to only have the service registration and not service start?